variables.tf

```diff
@@ -41,11 +41,12 @@ variable "vpc_id" {
 
 variable "subnet_ids" {
   description = "ID of subnets for the RDS subnet group."
-  type        = "list"
+  type        = list(string)
```

Comment:
I would also suggest adding type info to _all_ variables when we are at it. So `type = string`, `bool`, `number` etc where relevant.